### PR TITLE
Feat/config list

### PR DESCRIPTION
**Pull request by domenicsim1**

### CODE REVIEW

File: pkg/cmd/config/list/list.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+		cmd.Println(string(data))
+	case constants.OutputFormatBasic:
+		for _, key := range configFile.AllKeys() {
+			cmd.Println(configFile.GetString(key))
+		}
+	case constants.OutputFormatTable:
+		t := output.NewTable(cmd.OutOrStdout())
+		t.AddRow(output.Bold("KEY"), output.Bold("VALUE"))
+		for _, key := range configFile.AllKeys() {
+			t.AddRow(key, configFile.GetString(key))
```
Comment (borland) [inline pkg/cmd/config/list/list.go:83]: This is going to print the API key directly to the screen.
I feel like we should print ****s instead for this (same deal with JSON), but I'm not sure if I'm over-thinking it. 

Right now, the API key is trivially readable from the JSON file, so printing it to the screen isn't much worse -- but we have discussed things like putting the API key in the windows/macos keystore, and printing those values isn't something you'd want to be doing normally.

Would be good to get @jbristowe 's opinion on this one.

**Comment by borland** [comment]
PR looks good, I added a question about whether or not we should print API keys in config list. Deliberately neither approving nor blocking the PR, the team can resolve this question and go from there

File: pkg/cmd/config/list/list.go (hunk 1 above)
Comment (jbristowe) [reply pkg/cmd/config/list/list.go:83]: I'm inclined to agree here. Maybe we should do what they do with credit card numbers; show the last 4 digits and print `*` for the rest.

Comment (domenicsim1) [reply pkg/cmd/config/list/list.go:87]: As already said. It's still plain text in the json and the `config get apikey` command will still return the text value. I hid it anyway in the list command, as I could see "accidental" exposing the  key here as an issue as where doing it as `config get apikey` is it explicit.

**Review by jbristowe — APPROVED** [review]

**Pull request merged**